rails_app/app/adapters/graphite.rb

```diff
@@ -0,0 +1,60 @@
+require 'net/http'
+require 'json'
+require 'graphite-api'
+
+class GraphiteAdapter
+    def initialize
+        @graphite_host = Settings.graphite.host
+        @default_period = Settings.graphite.default_get_metric_time
+        @api = GraphiteAPI.new(
+            graphite: "#{@graphite_host}:#{Settings.graphite.port}",
+            prefix: Settings.graphite.metric_prefix
+        )
+    end
+
+    def send_raw(metric, value = nil, time = nil)
+        if !metric.is_a?(Hash)
```

Comment:
```metric = { metric => value } unless metric.is_a?(Hash)```

Reply:
I think this is counter intuitive to read that way?